compat/v0/compat_test.go

```diff
@@ -137,30 +137,28 @@ var _ = Describe("Compat V0", func() {
 
 		v1, err := v0.V1TxParamsFromTx(ctx, params, bindings, pubkey, store)
 		Expect(err).ShouldNot(HaveOccurred())
-		// Check that redis mapped the hashes correctly
-		hash := v1.Tx.Hash.String()
-		// btc txhash mapping
-		keys, err := client.Keys("*").Result()
 
 		// should have a key for the utxo
 		utxo := params.Tx.In.Get("utxo").Value.(v0.ExtBtcCompatUTXO)
-		keys, err = client.Keys(utxo.TxHash.String() + "_" + utxo.VOut.Int.String()).Result()
+		keys, err := client.Keys(utxo.TxHash.String() + "_" + utxo.VOut.Int.String()).Result()
 		Expect(err).ShouldNot(HaveOccurred())
 		Expect(len(keys)).Should(Equal(1))
 
+		// Check that redis mapped the hashes correctly
+		hash := v1.Tx.Hash.String()
 		storedHash, err := client.Get(keys[0]).Result()
 		Expect(err).ShouldNot(HaveOccurred())
-
 		Expect(storedHash).Should(Equal(hash))
 
 		// btc txhash mapping
 		keys, err = client.Keys("*").Result()
 
 		// v0 hash should have a mapping in the store
-		Expect(keys).Should(ContainElement("npiRyatJm8KSgbwA/EqdvFclMjfsnfrVY2HkjhElEDk="))
-
-		// v1 hash should be correct
-		Expect(hash).Should(Equal("YlkYzfTTCcptfS4bYdxnrXXNMv-C_6Y1UzWwi_wOrGI"))
```

Comment:
Any chance we can maintain a v1 hash correctness check?

Reply:
Yep, added.